pom.xml

```diff
@@ -11,7 +11,7 @@
 	</parent>
 
 	<artifactId>spring-cloud-gcp</artifactId>
-	<version>1.0.0.M1</version>
+	<version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
Can we also add back spring-cloud-gcp-examples here?

Reply:
I would prefer "No". IMHO.

I really don't like this manual tasks during release process. We need to have some automatic tool, indeed.

Reply:
Updated the PR with the samples module. Next time around, we will make the changes in the build script to not build the samples module so that we don't have to do this manual process.

Reply:
Sounds good @sobychacko !

The same is applied for the `docs` module as well.
See my comment: https://github.com/spring-cloud/spring-cloud-gcp/issues/170#issuecomment-341553552

Also need to figure out what to do with JavaDocs. But that's already different story and has to be addressed in the separate issue/PR.